src/grid.rs

```diff
@@ -1,38 +1,29 @@
 use macroquad::rand;
 
-pub const GRID_WIDTH: usize = 32;
-pub const GRID_HEIGHT: usize = 32;
-pub const CELL_SIZE: f32 = 30.0;
-
-#[derive(Copy, Clone, PartialEq, Default, Debug)]
+#[derive(Copy, Clone, PartialEq, Debug)]
 pub enum CellState {
-    #[default]
     Dead,
     Alive,
 }
 
-pub type Grid = [[CellState; GRID_WIDTH]; GRID_HEIGHT];
+pub type Grid = Vec<Vec<CellState>>;
```

Comment:
This should be a struct, and we can move the functions that receive it to its impl

Reply:
Done